rosgraph_msgs/msg/Clock.msg

```diff
@@ -0,0 +1,4 @@
+# roslib/Clock is used for publishing simulated time in ROS.
```

Comment:
copy-n-paste ?

Reply:
Yes, copied 1-to-1 from ROS 1.

Reply:
Please update the comment as there is no such thing as roslib/Clock in ROS 2. And this doesn't exist in ROS1 since C-Turtle either from what I can tell

Reply:
Done: ae2776765040d4d7c09cb948f9b1b23e6008c75e